python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.2.3
+
+### Bug fixes
+* When `if_exists` is set to `replace` in Dataframe operator, replace the table rather than append. This change fixes a regression on the Dataframe operator which caused it to append content to an output table instead of replacing.  [#1260](https://github.com/astronomer/astro-sdk/issues/1260)
+* Support `database` kwarg in `PostgresHook`. Since schema is renamed to database in airflow as per this [PR](https://github.com/apache/airflow/pull/26744), users can now set database instead of schema. [#1276](https://github.com/astronomer/astro-sdk/pull/1276)
```

Comment:
I think we should add something similar to "Pass the table metadata database value to the underlying airflow Postgres hook correctly" also do we need the Airflow PR link here @phanikumv @sunank200 ?

Reply:
Thats right- we arent doing any change to PostgresHook in this PR. Lets remove the airflow PR link and reword this

Reply:
But from user's perspective, the thing that would change is that we support `database` kwargs in `PoostgresHook` right?

Reply:
earlier we were database kwargs value to `self.table.metadata.schema` which was not correct

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1289/commits/166a8f7709641cb715f98b6a9bc54932a53192eb)